site/kubernetes.md

```diff
@@ -0,0 +1,90 @@
+## Configuring Oracle WebLogic Server Kubernetes Operator
+
+The Extract Domain Resource Tool can be used to create a domain resource file for use with the Oracle WebLogic Server Kubernetes Operator. This allows the domain configuration and the Kubernetes container configuration to be specified in a single model file.
+
+The Extract Domain Resource Tool is available with WDT releases 1.7.0 and later.
+
+More information about the Oracle WebLogic Server Kubernetes Operator can be found [here](https://github.com/oracle/weblogic-kubernetes-operator).
```

Comment:
More information about the Oracle WebLogic Server Kubernetes Operator can be found [here](https://github.com/oracle/weblogic-kubernetes-operator). Do you prefer to send your readers to the operator repository versus the docs, https://oracle.github.io/weblogic-kubernetes-operator?